templates/default/start-livy.sh.erb

```diff
@@ -14,6 +15,8 @@ if [ -f $PID_FILE ] ; then
  fi
 fi
 
+HADOOP_CLASSPATH=$($HADOOP_HOME/bin/hadoop classpath)
```

Comment:
@kouzant in most other cookbooks we use the `--glob` flag. Do you know why and also why it's not needed here?

Reply:
The `--glob` flag will expand `*` to absolute paths for systems that do not support expansion in their classpath. afaik it makes no difference in the platforms we target